pages/common/kustomize.md

```diff
@@ -0,0 +1,20 @@
+# kustomize
+
+> Kustomize is a tool to easily deploy resources for Kubernetes.
+> More information: <https://github.com/kubernetes-sigs/kustomize>.
+
+- Create kustomization.yaml file with resources:
+
+`kustomize create --resources deployment.yaml,service.yaml --namespace staging`
+
+- Build a kustomization.yaml file and deploy it with kubectl:
+
+`kustomize build . | kubectl apply -f -`
+
+- Edit kustomization.yaml file to set image:
```

Comment:
I would rather use the description from the command itself: `Sets images and their new names, new tags or digests in the kustomization file` 🤔

Reply:
You're right, to be consistent with the command, I will comment command with: 
> Set image in the kustomization file

And I will change other description with the same expression "in the kustomization file"